src/test/java/org/opensearch/security/multitenancy/test/TenancyMultitenancyEnabledTests.java

```diff
@@ -92,7 +92,7 @@ public void testMultitenancyDisabled_endToEndTest() throws Exception {
 
         final HttpResponse updateMutlitenancyToDisabled = nonSslRestHelper().executePutRequest(
             "/_plugins/_security/api/tenancy/config",
-            "{\"multitenancy_enabled\": \"false\"}",
+            "{\"multitenancy_enabled\": false}",
```

Comment:
these tests were supplying a string type but the multi-tenancy api validator expected a boolean type. my changes fixed it by introducing an additional case for handling BOOLEANs and hence this test failed for string type. CHanging it to boolean fixed it, which is correct way for this param.